.golangci.yml

```diff
@@ -18,8 +18,19 @@ linters:
     - ineffassign
     - misspell
     - unconvert
+    - typecheck
     - unused
+    # - staticcheck
+    - bidichk
+    - durationcheck
+    - copyloopvar
     - whitespace
+    # - revive # only certain checks enabled
+    - durationcheck
```

Comment:
this one is repeated like this in upstream at this commit, figured I'll leave it for now